apps/landingPage/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@kiwicom/margarita-landing-page",
+  "private": true,
+  "version": "0.1.0",
+  "dependencies": {
+    "@kiwicom/orbit-components": "^0.28.2",
+    "@kiwicom/orbit-design-tokens": "^0.3.0",
+    "gatsby": "^2.0.105",
+    "gatsby-image": "^2.0.20",
+    "gatsby-plugin-flow": "^1.0.2",
+    "gatsby-plugin-manifest": "^2.0.15",
+    "gatsby-plugin-offline": "^2.0.22",
+    "gatsby-plugin-react-helmet": "^3.0.2",
+    "gatsby-plugin-sharp": "^2.0.19",
+    "gatsby-source-filesystem": "^2.0.8",
+    "gatsby-transformer-sharp": "^2.1.13",
+    "prop-types": "^15.6.2",
```

Comment:
I think this package also does not use anywhere.

Reply:
There were too many conflicts in yarn.lock

Reply:
which one? prop-types?